.github/workflows/run_tests.yaml

```diff
@@ -0,0 +1,53 @@
+name: Run test suite
+on: [push, pull_request]
+
+jobs:
+  check_version:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: [ 3.8 ]
+        os: [ ubuntu-latest ]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
```

Comment:
It looks like the version check here is incomplete -- all this job does is checkout code and install Python.

Reply:
Added the same check as the other workflow